### DISCUSSION TITLE
[Feature] Add ability to import/Create Albums mapping to SMB share mapping

### DISCUSSION THREAD

**Discussion by ChinmayAmin** [immich-app/immich#8162] [Feature Request]
### The feature

Idk if this is possible some way but would be cool if i can mount a path into immich and go under albums and point to that path. I do a read only for photos atm but i couldn't do this for albums. Would be cool.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4279
